go.mod

```diff
@@ -30,6 +30,11 @@ require (
 
 replace (
 	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
+
+	// Required indirectly by manifestival.
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20190924102528-32369d4db2ad
```

Comment:
This surprises me. I would consider this a manifestival bug.

Reply:
I would've looked at that next, but I wanted to fix the immediate bug of not being able to run update-deps at all :joy:

Reply:
Could also be an artifact of manifestival using dep and the operator now using go mod.

Reply:
Okay, I dug it out. Proper fix in https://github.com/manifestival/client-go-client/pull/2 and #93